pkg/segment/query/processor/wherecommand.go

```diff
@@ -72,7 +72,6 @@ func (p *whereProcessor) Process(iqr *iqr.IQR) (*iqr.IQR, error) {
 				log.Errorf("where.Process: cannot evaluate expression; err=%v", err)
 				p.loggedEvalError = true
 			}
-			continue
 		}
```

Comment:
Hmm, I tried to simulate a case where this would happen, but since, now we don't throw errors on `null` and even when there is any conversion issues for numbers, we are not throwing an error now. 
So the only case that I see an error happening right now, is when there is a syntax issue, which is taken care by the parser.

Reply:
So I am thinking instead of logging should I return this error? As I think this to be a rare case?